### ISSUE TITLE
Observers need a way to latejoin rounds

### ISSUE THREAD

**Issue by metalgearsloth** [space-wizards/space-station-14#4780]
Currently if you accidentally get observer'd you're kinda stuck there until next round or a ghost role pops up. Ideally this would only be for observers and not regular ghosts.

**Comment by Zumorica** [comment]
I don't think this is something we want to allow players to do just like that. 
Observing a round should be a conscious option that leaves you out of the round for the most part, otherwise someone could observe, learn who the traitor/antag is, then join in and ruin the round for everyone.

**Comment by mirrorcult** [comment]
if you want observers to be able to join the round that's why ghost roles exist

**Comment by metalgearsloth** [comment]
Then we need a way to not accidentally force players as observers when force starting a round and increase the lobby timer

**Comment by Zumorica** [comment]
> If the lobby timer is increased, make it autostart when everyone has pressed Ready Up, as well.

Doesn't sound like a good idea, to be honest.

**Comment by metalgearsloth** [comment]
Whatever the solution observing should be a conscious decision. Right now we still get complaints that people have accidentally started observing a round for one reason or another.

**Comment by ShadowCommander** [comment]
We don't have a prompt when clicking observe like some SS13 servers have. Just asking "Are you sure you want to observe? You cannot join the round as a normal crew member if you continue." should alleviate most of the problem where players accidentally hit observe instead of join.

I'll make a PR for the popup later this week if no one else does first.
